packages/plugin-types/src/package_manager_controller.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * TODO: use the latest execa.
+ * execa v8 doesn't support commonjs, so we need to use the types from v5
```

Comment:
Isn't this going to cause problems since cli-core uses execa v 8?

Reply:
That was my concern as well, but the tests look fine.

Is there anything else we should check?

Reply:
I guess we'll cross that bridge if/when we get there :)